README.md

```diff
@@ -47,9 +48,12 @@ brew install hashicorp/tap/terraform
 brew install hashicorp/tap/vault
 brew install hashicorp/tap/waypoint
 
-brew cask install vagrant
+# Casks
+brew install hashicorp-boundary-desktop
+
+brew install vagrant
 ```
-* Note: Vagrant is available as a cask from Homebrew core. It is packaged as a cask to include required dependencies of Vagrant.
+* Note: Vagrant is available as a cask from Homebrew core and not provided by this tap.
```

Comment:
not sure: is it Vagrant only or Boundary Desktop as well?

Reply:
Updated this documentation because the cask differentiation is no longer relevant since we now have a cask! Vagrant will likely be included in this tap's casks once we can properly support casks